applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_server.c

```diff
@@ -358,7 +357,12 @@ static void stream_recv_cb(struct bt_bap_stream *stream, const struct bt_iso_rec
 static void stream_sent_cb(struct bt_bap_stream *stream)
 {
 	/* Unicast server/CIS headset only supports one source stream for now */
-	ERR_CHK(bt_le_audio_tx_stream_sent(0));
+	struct stream_index stream_idx = {
+		.lvl1_idx = 0,
```

Comment:
This looks odd. IF this headset is part of CIG 1, then here it sets 0 ?

Reply:
The unicast_server has no concept of what CIG it is a part of, but we still need this stream to have a 3D index,

Reply:
Add stream_index_get function